setup.py

```diff
@@ -570,26 +570,46 @@ def determine_gl_flags():
 
 def determine_sdl2():
     flags = {}
+    # This is redundant; the only time determine_sdl2 is called is preceded by
+    # verification that this condition is already satisfied
     if not c_options['use_sdl2']:
         return flags
 
+    # Check the environment for a Kivy SDL2 path (all platforms) or for Include
+    # and Lib (intended to catch Windows installation, see #4941)
```

Comment:
Generally it's not a good idea to add comments that just repeat what we can understand from looking at the code. I think most of these comments above and below can be removed. 

The idea of comments is to give the high level intent of the coder when it's not obvious from just looking at the code by an experienced coder. Generally python code is clear enough to need very few comments to an experienced python developer. So comments should only be added when that higher level intent is not clear. I would say that only the comment `# no pkgconfig info, or we want to use a specific sdl2 path, so perform manual configuration` is adding that as it explains why we are doing this stuff. Pretty much all the other comments should be removed.

You can see https://en.wikipedia.org/wiki/Comment_(computer_programming)#Code_description.